### PR TITLE
Topic1 ex 4

### PR DESCRIPTION
**Pull request by julianpoma**
Disregard the CSS, I copied the main structure of the folders from the previous ex and forgot to delete it

### CODE REVIEW

File: 01-JS-DOM-APIs/4/js/javascript.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+function getrepos () {
+    var xhttp = new XMLHttpRequest();
```
Comment (maximilianotulian) [inline 01-JS-DOM-APIs/4/js/javascript.js]: Try to reuse the function we just created in the previous exercise

File: 01-JS-DOM-APIs/4/js/javascript.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+function getrepos () {
+    var xhttp = new XMLHttpRequest();
+    let query = document.getElementById('search').value;
+    let basic = "https://api.github.com/search/repositories?q="
```
Comment (maximilianotulian) [inline 01-JS-DOM-APIs/4/js/javascript.js]: Remember the single quotes in javascript

File: 01-JS-DOM-APIs/4/js/javascript.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    let query = document.getElementById('search').value;
+    let basic = "https://api.github.com/search/repositories?q="
+    let url = basic.concat(query);
+
+    xhttp.open("GET", url, true);
+    xhttp.onreadystatechange = function() {
+
+        if (this.readyState == 4 && this.status == 200) {
+
+            var arrayResponse = JSON.parse(xhttp.responseText);
```
Comment (maximilianotulian) [inline 01-JS-DOM-APIs/4/js/javascript.js]: es6 for win! Did you read about hoisting?

File: 01-JS-DOM-APIs/4/js/javascript.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+            for (var i=0; i<arrayResponse.items.length; i++) {
+
+                let newLi = document.createElement("li");
```
Comment (maximilianotulian) [inline 01-JS-DOM-APIs/4/js/javascript.js]: Try to be consistent, you're using `var` and `let`. Let's try to use only let

File: 01-JS-DOM-APIs/4/js/javascript.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+                let data = document.createTextNode(arrayResponse.items[i].full_name);
+                newLi.appendChild(data);
+                document.getElementById('variable-text').appendChild(newLi);
+            }
+
+        }
+    };
+
+    xhttp.onerror = function () {
+            document.getElementById("variable-text").style.color = red;
```
Comment (maximilianotulian) [inline 01-JS-DOM-APIs/4/js/javascript.js]: **Minor**: Remove the additional tab

File: 01-JS-DOM-APIs/4/js/javascript.js (hunk 1 above)
Comment (julianpoma) [inline 01-JS-DOM-APIs/4/js/javascript.js]: Ok !

File: 01-JS-DOM-APIs/4/js/javascript.js (hunk 2 above)
Comment (julianpoma) [inline 01-JS-DOM-APIs/4/js/javascript.js]: Ok!

**Pull request merged**